### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by Aishwarya-Hariharan-Iyer**
Updated the UG to contain commands planned for v1.1 of DevEnable. 

Based on our team document linked [here](https://docs.google.com/document/d/1_iJyfbqPF-lV3i2GC0nB_dKrSA7zzxvQ7D_Smc7o-Oc/edit?usp=sharing).

Things to do:
- Picture of UI to be added
- Commands to be reviewed after implementation

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -14,27 +16,27 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest `deveneable.jar`.
```
Comment (ConradLew) [inline docs/UserGuide.md:19]: is there a spelling error here?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -45,106 +47,170 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+  e.g. if the command specifies `client -t PROJECT_ID n/CLIENT_NAME [p/CLIENT_CONTACT_NUMBER] [e/CLIENT_EMAIL]`, 
+  `client -t PROJECT_ID n/CLIENT_NAME [e/CLIENT_EMAIL] [p/CLIENT_CONTACT_NUMBER]` are both acceptable.
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
+* If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence 
+  of the parameter will be taken.<br>
   e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`)
```
Comment (ConradLew) [inline docs/UserGuide.md:66]: 'exit' may not be relevant to our commands

**Review by ConradLew — CHANGES_REQUESTED** [review]
LGTM. Minor nits of spelling, pull request should probably be made from branch 'update-userguide' instead of master to align with forking workflow

File: docs/UserGuide.md [hunk 3]
```diff
@@ -14,27 +16,27 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-   ![Ui](images/Ui.png)
+4. Double-click the file to start the app. 
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will 
+open the help window.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`list`** : Lists all project.
```
Comment (crvstalphua) [inline docs/UserGuide.md:29]: i think project should be plural

File: docs/UserGuide.md [hunk 4]
```diff
@@ -14,27 +16,27 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`project -a`**`n/Orbital` : Adds a project named `Orbital` to the Address Book.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`project -d`**`3` : Deletes the 3rd project shown in the current list.
 
    * **`clear`** : Deletes all contacts.
```
Comment (crvstalphua) [inline docs/UserGuide.md:35]: should be all projects?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -14,27 +16,27 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`list`** : Lists all project.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`project -a`**`n/Orbital` : Adds a project named `Orbital` to the Address Book.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`project -d`**`3` : Deletes the 3rd project shown in the current list.
```
Comment (crvstalphua) [inline docs/UserGuide.md:33]: wanted to check, does 3 refer to the project_id?

File: docs/UserGuide.md [hunk 6]
```diff
@@ -45,106 +47,170 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+*  `client -e 1 n/Amy` Edits the client tagged to project with `PROJECT_ID` 1 with new `CLIENT_NAME` Amy.
+*  `client -e 2 n/Bob p/91234567` Edits the client tagged to project with `PROJECT_ID` 2 with new `CLIENT_NAME` Amy 
+   and `CLIENT_CONTACT_NUMBER` 91234567.
+*  `client -e 3 n/Charlie e/charlie@gmail.com` Edits the client tagged to project with `PROJECT_ID` 3 with new 
+   `CLIENT_NAME` Charlie and `CLIENT_EMAIL` charlie@gmail.com.
+*  `client -e 2 n/Dave p/91111111 e/dave@gmail.com`
```
Comment (crvstalphua) [inline docs/UserGuide.md:161]: I think u missed out the explanation for this line. 
```suggestion
*  `client -e 2 n/Dave p/91111111 e/dave@gmail.com` Edits the client tagged to project with `PROJECT_ID` 2 with new `CLIENT_NAME` Dave, `CLIENT_CONTACT_NUMBER` 91111111 and `CLIENT_EMAIL` dave@gmail.com.
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -175,18 +244,25 @@ _Details coming soon ..._
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains 
+the data of your previous AddressBook home folder.
```
Comment (crvstalphua) [inline docs/UserGuide.md:248]: i think AddressBook should be DevEnable?

**Review by crvstalphua — CHANGES_REQUESTED** [review]
LGTM. Other than the minor typos that I've commented on.

File: docs/UserGuide.md (hunk 1 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/UserGuide.md:19]: Noted. Thank you!

File: docs/UserGuide.md (hunk 2 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/UserGuide.md:66]: Noted and removed. Thank you!

File: docs/UserGuide.md (hunk 3 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/UserGuide.md:29]: Changed. Thank you!

File: docs/UserGuide.md (hunk 4 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/UserGuide.md:35]: Changed. Thank you!

File: docs/UserGuide.md (hunk 5 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/UserGuide.md:33]: Yes, it does! We generate a unique PROJECT_ID upon the addition of each project. At the moment, that would coincide with the project's position on the list. However, if upon implementing the command, we find a better approach/nomenclature, we can make changes to the UG accordingly.

File: docs/UserGuide.md (hunk 6 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/UserGuide.md:161]: I had left the explanation out as it seemed redundant given the previous two commands, but I can include it in a new commit for better consistency...

File: docs/UserGuide.md (hunk 7 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/UserGuide.md:248]: Changed. Thank you!

**Comment by Aishwarya-Hariharan-Iyer** [comment]
Issued a new PR from update-userguide branch as suggested.

**Pull request closed without merging**